docs/quality-assurance/test-tooling.md

```diff
@@ -30,3 +32,65 @@
 |Jmeter|Load test functional behaviour and measure performance|https://jmeter.apache.org/||
 |REST-assured|Testing and validating REST services|http://rest-assured.io/||
 |swagger-request-validator|A Java library for validating request/responses against a OpenAPI / Swagger specification|https://bitbucket.org/atlassian/swagger-request-validator||
+
+## Browser Automation - what tools to use?
+This section will cover the browser automation tools we use at the UKHO and the recommended uses for each one. 
+
+The 3 recommended tools for Browser Automation (UI, e2e and Integration) at the UKHO are:
+
+### Cypress
+Cypress is a front end testing tool built for testing web apps. Cypress test code is executed directly within the browser as JavaScript, meaning there are no language or driver bindings and the tests can execute much faster and with more reliability. It also has full dom and network traffic recording to aid debugging within the Cypress Test Runner. 
+
+Cypress, however, does have its limitations. It has fundamently been created to test your application and your application only, and works best if the Application-Under-Test is a Single Page Application or a Multi Page Application in the same domain that does not utilise pop-ups or new tabs / windows.
```

Comment:
Does this bit relate to only mult- .. or single-pgae apps? "that does not utilise pop-ups or new tabs / windows." - if the former then it's ok.